atmosphere/um_env.yaml

```diff
@@ -0,0 +1,50 @@
+AINITIAL: ''
+ASTART: INPUT/pre-industrial.astart
+AUSCOM_CPL: 'true'
+DATAW: '{work_path}'
+FASTRUN: 'true'
+IDEALISE: ''
+PAREXE: parexe
+PRINT_STATUS: PrStatus_Oper
+RPSEED: ''
+RUNID: HI-24
+TYPE: NRUN
+UM_ATM_NPROCX: '16'
+UM_ATM_NPROCY: '12'
+UM_NAM_MAX_SECONDS: '300'
+UM_NPES: '192'
+UM_SECTOR_SIZE: '2048'
+UM_STDOUT_FILE: atm.fort6.pe
+VN: '7.3'
+
+# Config input paths
+UNIT01: ''
+UNIT02: prefix
+UNIT04: STASHC
+UNIT05: namelists
+UNIT07: PI-01.out2
+UNIT08: /dev/null
+UNIT09: CONTCNTL
+UNIT10: xhist
+UNIT11: ihist
+UNIT12: thist
+UNIT14: errflag
+UNIT15: ''
+UNIT58: ''
+UNIT22: INPUT/STASHmaster
+UNIT57: INPUT/spec3a_sw_hadgem1_6on
+UNIT80: INPUT/spec3a_lw_hadgem1_6on
+STASETS_DIR: INPUT/stasets
+VERT_LEV: INPUT/vertlevs_G3
+
+# Ancillary files
+ARCLBIOG: INPUT/biogenic_351sm.N96L38
+BIOMASS: INPUT/Bio_1849_2015_ESM1.anc
+CHEMOXID: INPUT/sulpc_oxidants_N96_L38
+DMSCONC: INPUT/DMS_conc.N96
```

Comment:
Or input lines for these four?

Reply:
These ones were also added in #39 and so aren't showing up as being added here.